core/pom.xml

```diff
@@ -30,6 +30,12 @@
   </developers>
   <properties>
     <project.rootdir>${project.basedir}/../</project.rootdir>
+    <coverage.complexity>60%</coverage.complexity>
+    <coverage.line>70%</coverage.line>
+    <coverage.instruction>68%</coverage.instruction>
+    <coverage.branch>52%</coverage.branch>
+    <coverage.method>73%</coverage.method>
+    <coverage.class>87%</coverage.class>
```

Comment:
orchestration tests don't count towards core coverage, therefore I separated the values into individual modules so these bad limits on the core module don't impact the other modules